locations/spiders/sportscraft.py

```
@@ -1,39 +1,48 @@
-from locations.dict_parser import DictParser
+from locations.categories import Categories, apply_category
 from locations.hours import OpeningHours
+from locations.items import Feature
+from locations.json_blob_spider import JSONBlobSpider
 from locations.pipelines.address_clean_up import clean_address
 
 
-class SportscraftSpider(Spider):
+class SportscraftSpider(JSONBlobSpider):
     name = "sportscraft"
     item_attributes = {"brand": "Sportscraft", "brand_wikidata": "Q7579966"}
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)